crates/bevy_ecs/src/schedule/graph/mod.rs

```diff
@@ -82,24 +82,24 @@ pub(crate) fn row_col(index: usize, num_cols: usize) -> (usize, usize) {
 }
 
 /// Stores the results of the graph analysis.
-pub(crate) struct CheckGraphResults {
+pub(crate) struct CheckGraphResults<Id: GraphNodeId> {
```

Comment:
Very minor nit, but you used `N: GraphNodeId` above and `Id: GraphNodeId` here and it might be good to use a consistent type parameter name.

Reply:
Aligned them all to `N`.